hydra-node/src/Hydra/Node/Network.hs

```diff
@@ -130,7 +131,7 @@ withNetwork tracer configuration callback action = do
         withFlipHeartbeats $
           withReliability (contramap Reliability tracer) messagePersistence me otherParties $
             withAuthentication (contramap Authentication tracer) signingKey otherParties $
-              withOuroborosNetwork (contramap Network tracer) localhost peers
+              withOuroborosNetwork (contramap Network tracer) HydraNetworkConfig{protocolVersion = pVersion, localHost, remoteHosts = peers} (const $ pure ())
```

Comment:
Must: send a server output when a handshake was not possible

This is specified in the feature description and the api.yaml promises it, but `const $ pure ()` is obviously not doing anything.